src/main/java/net/sf/jabref/gui/shared/SharedDatabaseUIManager.java

```diff
@@ -98,7 +98,7 @@ public void listen(SharedEntryNotPresentEvent event) {
      * @param dbmsConnectionProperties Connection data
      * @param raiseTab If <code>true</code> the new tab gets selected.
      */
-    public void openNewSharedDatabaseTab(DBMSConnectionProperties dbmsConnectionProperties)
+    BasePanel openNewSharedDatabaseTab(DBMSConnectionProperties dbmsConnectionProperties)
```

Comment:
Please comment why this is package private? Testing is not the reason. Maybe this should be private or public?